app/views/bookings/_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= simple_form_for(restaurant) do |f| %>
```

Comment:
isn't it
`<%= simple_form_for [@user, @boat, @booking] do |f| %>` ?
We don't have a restaurant 😄 
And the bookings belongs to users and boats

Reply:
Like when we needed to make a new dose for mr cocktail. The simple form was `[@cocktail, @dose]`

Reply:
oops. I blindly copied and pasted the code. Thank you for having noticed it. I corrected it.